e2e-test/cypress/integration/payload_validation_for_assign_project_spec.js

```diff
@@ -0,0 +1,75 @@
+describe("Project Assignee data validation", function() {
```

Comment:
I think there is no extra file needed for the validation tests because already I think the tests already exist in the other files.